packages/generator/src/graphql-modules/index.ts

```diff
@@ -44,13 +43,12 @@ export class GenerateModules extends Generators {
             (item) => item.name === field.name,
           );
           if (
-            dataField?.kind === 'object' &&
-            dataField.relationFromFields.length > 0
+            dataField?.kind === 'object'
           ) {
             if (!modules.includes(dataField.type + 'Module') && model.name !== dataField.type) {
               modules.push(dataField.type + 'Module');
-              imports += `import { ${dataField.type}Module } from '../${dataField.type}/${dataField.type}.module';
-              `;
+            }
+            if (model.name !== dataField.type) {
               extendsTypes += `extend type ${dataField.type} {`;
               models
                 .find((item) => item.name === dataField.type)
```

Comment:
In this line, we extend all relations fields with this model.  so if we get another field for the model we added before, we must skip it or will get a duplicate like this 
![image](https://user-images.githubusercontent.com/4535142/99314895-7c2e8500-286a-11eb-9b4a-6bc2f6f5584b.png)